openapi.yaml

```diff
@@ -734,35 +734,36 @@ components:
         type: string
         minimum: 9
         maximum: 200
-    knittingProcessParam:
-      name: knittingProcess
+    fabricProcessParam:
+      name: fabricProcess
       in: query
       description: |
```

Comment:
As the param is now required, it should be marked as such and moved up the fields list, next to other required fields
![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/c43c0e72-7fda-4cf9-942b-74dfce4b3254)